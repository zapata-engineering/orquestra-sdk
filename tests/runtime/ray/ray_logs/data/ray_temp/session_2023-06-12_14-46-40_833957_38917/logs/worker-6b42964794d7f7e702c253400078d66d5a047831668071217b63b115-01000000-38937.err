:job_id:01000000
:task_name:_workflow_task_executor_remote
2023-06-12 14:46:55,342	INFO task_executor.py:78 -- Task status [RUNNING]	[wf.wf_using_python_imports.8a4d9e7@invocation-1-task-add-with-log]
{"timestamp": "2023-06-12T12:46:55.381839+00:00", "level": "INFO", "filename": "_example_wfs.py:283", "message": "hello, there!", "wf_run_id": "wf.wf_using_python_imports.8a4d9e7", "task_inv_id": "invocation-1-task-add-with-log", "task_run_id": "wf.wf_using_python_imports.8a4d9e7@invocation-1-task-add-with-log.f7e22"}
:task_name:_workflow_task_executor_remote
2023-06-12 14:47:42,508	INFO task_executor.py:78 -- Task status [RUNNING]	[wf.wf_using_python_imports.8a4d9e7@orquestra.sdk._ray._build_workflow._make_ray_dag_node.locals._ray_remote]
:task_name:_workflow_task_executor_remote
2023-06-12 14:47:42,520	INFO task_executor.py:78 -- Task status [RUNNING]	[wf.wf_using_python_imports.8a4d9e7@orquestra.sdk._ray._build_workflow.make_ray_dag.locals.handle_data_aggregation_error]
