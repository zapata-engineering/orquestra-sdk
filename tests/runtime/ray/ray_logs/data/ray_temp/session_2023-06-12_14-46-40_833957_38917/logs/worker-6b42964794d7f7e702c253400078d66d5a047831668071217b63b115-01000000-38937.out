:job_id:01000000
:task_name:_workflow_task_executor_remote
:task_name:_workflow_task_executor_remote
:task_name:_workflow_task_executor_remote
